syntax = "proto3";

package communication;

service FormulaEvaluator {
  rpc EvaluateFormula (FormulaRequest) returns (FormulaResponse);
}

message FormulaRequest {
  string formula = 1;
  string data = 2;
}

message FormulaResponse {
  string result = 1;
}
